app/src/org/commcare/activities/AdvancedActionsActivity.java

```diff
@@ -45,11 +45,12 @@
     private final static int WIFI_DIRECT_ACTIVITY = 1;
     private final static int DUMP_FORMS_ACTIVITY = 2;
 
-    public final static int RESULT_DATA_RESET = RESULT_FIRST_USER + 1;
-    public final static int RESULT_FORMS_PROCESSED = RESULT_FIRST_USER + 2;
+    public final static int RESULT_DATA_RESET = CommCareHomeActivity.RESULT_RESTART + 1;
+    public final static int RESULT_FORMS_PROCESSED = CommCareHomeActivity.RESULT_RESTART + 2;
```

Comment:
Other issue was that these result codes conflicted with `CommCareHomeActivity.RESULT_RESTART` resulting in incorrect processing upon return from `AdvancedActionsActivity`